exonum-java-binding-core/src/main/java/com/exonum/binding/blockchain/TransactionResult.java

```diff
@@ -21,15 +21,14 @@
 import com.exonum.binding.transaction.TransactionExecutionException;
 import com.google.auto.value.AutoValue;
 import com.google.common.base.Strings;
-import java.util.Optional;
 import java.util.OptionalInt;
 import javax.annotation.Nullable;
 
 /**
  * Returns a result of the transaction execution. This result may be either a success, or an error,
  * if execution has failed.
  * Errors might be either service-defined or unexpected. Service-defined errors consist of an error
- * code and an optional description. Unexpected errors consist of an optional description only.
+ * code and an optional description. Unexpected errors include only a description.
```

Comment:
`only an optional description`?

Reply:
I decided to skip it here because it is mentioned above that a description can be optional.